pkg/serviceaccounts/oauthclient/oauthclientregistry_test.go

```
@@ -816,8 +852,9 @@ func TestParseModelsMap(t *testing.T) {
 			},
 		},
 	} {
-		if !reflect.DeepEqual(test.expected, parseModelsMap(test.annotations, decoder)) {
-			t.Errorf("%s: expected %#v, got %#v", test.name, test.expected, parseModelsMap(test.annotations, decoder))
+		models, _ := parseModelsMap(test.annotations, decoder)
```

Don't ignore the errors.